synapse/storage/databases/main/keys.py

```diff
@@ -115,13 +121,7 @@ def store_server_verify_keys(self, from_server, ts_added_ms, verify_keys):
             # param, which is itself the 2-tuple (server_name, key_id).
             invalidations.append((server_name, key_id))
 
-        def _invalidate(res):
-            f = self._get_server_verify_key.invalidate
-            for i in invalidations:
-                f((i,))
-            return res
```

Comment:
`simple_upsert_many_txn` returns `None`, so the input to the callback (`res`) was always `None`.

There's no reason to store this and return it, it makes it more confusing.